src/repr/Cargo.toml

```diff
@@ -16,6 +16,7 @@ harness = false
 chrono = { version = "0.4", features = ["serde"] }
 failure = "0.1.6"
 hex = "0.4.1"
+log = "0.4.5"
```

Comment:
Gets used [here](https://github.com/MaterializeInc/materialize/blob/0a3400e34e34072fb49b893d9a6cf02176265c9a/src/repr/scalar/datetime.rs#L689), which was a suggestion from the original interval PR.

Reply:
Eep, ok, I guess that's grandfathered in, but logging on the data plane is dangerous—and unlikely to be actionable.

Reply:
Yeah, ideally this would be covered by future error dataflow stuff, if it lives on. Generally, if we end up calling this lots it is going to stress out the independence of the worker threads.